superset-frontend/package.json

```diff
@@ -82,6 +82,7 @@
     "last 3 edge versions"
   ],
   "dependencies": {
+    "@apache-superset/core": "file:packages/superset-core",
```

Comment:
I think this is creating some troubles for me. UI won't boot up...

Reply:
What I know so far...
- nuking node_modules, `npm run prune`, restarts
- after this PR, running `npm install` changes the package-lock.json
- I can only get the UI to boot up on SHAs before this PR
- building before/after this in master, works before, fails after

Reply:
@mistercrunch I opened https://github.com/apache/superset/pull/34924 to update the referenced version of @apache-superset/core. I tested both dev and built modes:

```
superset init
npm ci
npm run build (serving from port 8088 OK)
npm run dev-server (serving from port 9000 OK)
```